tests/main/install-store/task.yaml

```diff
@@ -44,4 +44,12 @@ execute: |
     snap install --edge test-snapd-complexion
 
     echo "All snap blobs are 0600"
-    test "$( find /var/lib/snapd/{snaps,cache,seed/snaps}/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
+    test "$( find /var/lib/snapd/{snaps,cache}/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
```

Comment:
This is not enough for core 20, some snaps on /var/lib/snapd/snaps/ also have permissions 0700

google:ubuntu-core-20-64 .../tests/main/install-sideload# find /var/lib/snapd/snaps/ /var/lib/snapd/cache/  -type f -printf '%p %#m\n'
/var/lib/snapd/snaps/test-snapd-sh_2.snap.partial 0600
/var/lib/snapd/snaps/core20_284.snap 0700
/var/lib/snapd/snaps/core_8268.snap 0600
/var/lib/snapd/snaps/test-snapd-devmode_x1.snap 0600
/var/lib/snapd/snaps/basic_x1.snap 0600
/var/lib/snapd/snaps/test-snapd-tools_6.snap.partial 0600
/var/lib/snapd/snaps/pc-kernel_362.snap 0700
/var/lib/snapd/snaps/test-snapd-tools_x1.snap 0600
/var/lib/snapd/snaps/test-snapd-devmode_x2.snap 0600
/var/lib/snapd/snaps/basic-desktop_x1.snap 0600
/var/lib/snapd/snaps/pc_81.snap 0700
/var/lib/snapd/snaps/jq_6.snap.partial 0600
/var/lib/snapd/snaps/test-snapd-tools_x2.snap 0600
/var/lib/snapd/snaps/snapd_x1.snap 0700
/var/lib/snapd/snaps/test-snapd-rsync-core20_1.snap 0600
/var/lib/snapd/cache/7946b4de33a1b621a7c9f22a1463d858c55d09db7294b187527127ef1c71b126dc7d687f294d89f7c2eb63a644af982f 0600
/var/lib/snapd/cache/9f0ee5d3dd59ae4e46b04ab358778729a4b29992177cecf6511c070c0e67f67540e557f6a7c48e985e658c93c4bbb2f1 0600
/var/lib/snapd/cache/b25a38e0a01970659c840c675788f2143a274f7b08a99f3ee765831d9c19b502c1af1cefb0caced3105a07f0bb2cc6ab 0600

Reply:
@mvo5, is this expected or needs to be fixed?

Reply:
we should revert this file, now is failing the test on ubuntu 14.04: https://paste.ubuntu.com/p/WjSHrW2bTv/